### DISCUSSION TITLE
I was using Github for 1 months but got weird problem. Pls check

### DISCUSSION THREAD

**Discussion by Thikhonggioicode** [community/community#141206] [New to GitHub]
Can say the markdown file is emty but my file before using git add already got something inside
![image](https://github.com/user-attachments/assets/da0c05dc-05b1-42a8-9c3e-6e0f058a45e7)
This is what I write in 
![image](https://github.com/user-attachments/assets/dc68988f-313a-442d-94a4-f49afb4cfab4)
And when I do all the commit things 
![image](https://github.com/user-attachments/assets/815dd54b-b2ac-492a-bc85-b42cf69d22d7)
It still emty by somehow even when I try to do other way but still get the same result
![image](https://github.com/user-attachments/assets/e0ee55e1-97a0-477e-bfb0-0a51bb48f8a4)

**Comment by marlonwq** [comment]
Hi, I didn't find the repository in your profile, but this article might be useful: https://github.com/hackergrrl/art-of-readme.
And maybe using github desktop to make commits would help: https://github.com/apps/desktop.

**Comment by mctrinh** [comment]
Did you press Ctrl + S to save your README.md file?
